### ISSUE TITLE
APP Launch Password Protection [FTR]

### ISSUE THREAD

**Issue by fux0r2009** [austinhuang0131/barinsta#1122]
**Is your feature request related to a problem? Please describe.**
The App can easily be accessed by just launching the app 

**Describe the solution you'd like**
An option to make a password to protect the app and  is asked at every launch of the app 

**Additional context**
It would help make the app more private 

I hope you will Implement this feature, Thank you!

**Comment by Martin5001** [comment]
Biometrics support would be nice too

**Comment by fux0r2009** [comment]
> Biometrics support would be nice too

yes that would be a nice to have too but password protection is a must! hope dev will implement it :)
